### PR TITLE
update the contract addresses

### PR DESCRIPTION
**Pull request by poiresel**
Prior to this merge: npm package will need to have beta removed.

This change bumps the indexer to be the most recent package but keeps the existing contract address.

Updated the npm packages and contract addresses for Rinkeby and Mainnet

### CODE REVIEW

File: instant/contract-deployments.md [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
+| Indexer  | [`3.6.8`](https://www.npmjs.com/package/@airswap/indexer/v/3.6.8)  | [`0x10F6702447414cE1250Af5f7000D7c9A0f04E824`](https://rinkeby.etherscan.io/address/0x10F6702447414cE1250Af5f7000D7c9A0f04E824) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
+| Delegate | [`2.6.8`](https://www.npmjs.com/package/@airswap/delegate/v/2.6.8) | [`0xd393FB3f3062e208049FC75B26BFa5aB0890CBB3`](https://etherscan.io/address/0xd393FB3f3062e208049FC75B26BFa5aB0890CBB3)         | [`953956f`](https://github.com/airswap/airswap-protocols/commit/953956f308c65ec53d1f1b20d35f47fe04b936af) |
+| Wrapper  | [`3.6.8`](https://www.npmjs.com/package/@airswap/wrapper/v/3.6.8)  | [`0x33356FcCF3bdeb37f3CD3Dc5382Ae5BF4AAda26c`](https://rinkeby.etherscan.io/address/0x33356FcCF3bdeb37f3CD3Dc5382Ae5BF4AAda26c) | [`953956f`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
```
Comment (alicevhenshaw) [inline instant/contract-deployments.md]: github link doesnt match the displayed commit

File: instant/contract-deployments.md [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
-| Wrapper  | [`2.5.7`](https://www.npmjs.com/package/@airswap/wrapper/v/2.5.7)  | [`0x33356FcCF3bdeb37f3CD3Dc5382Ae5BF4AAda26c`](https://rinkeby.etherscan.io/address/0x33356FcCF3bdeb37f3CD3Dc5382Ae5BF4AAda26c) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
-| Types    | [`2.4.4`](https://www.npmjs.com/package/@airswap/types/v/2.4.4)    | [`0xe25b7504856bfb230b7c32BC87047479815cbc70`](https://rinkeby.etherscan.io/address/0xe25b7504856bfb230b7c32BC87047479815cbc70) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
+| Swap     | [`5.4.7`](https://www.npmjs.com/package/@airswap/swap/v/5.4.7)     | [`0xa03855b0695013cF331FbD91dd7cBc4C2211793f`](https://rinkeby.etherscan.io/address/0xa03855b0695013cF331FbD91dd7cBc4C2211793f) | [`953956f`](https://github.com/airswap/airswap-protocols/commit/953956f308c65ec53d1f1b20d35f47fe04b936af) |
```
Comment (alicevhenshaw) [inline instant/contract-deployments.md]: have the rinkeby addresses not changed?

File: instant/contract-deployments.md [hunk 3]
```diff
@@ -2,21 +2,21 @@
...
-| Types    | [`2.4.4`](https://www.npmjs.com/package/@airswap/types/v/2.4.4)    | [`0xef5ED6D6f1600C56c54c51841bCF97421CDdb52d`](https://etherscan.io/address/0xef5ED6D6f1600C56c54c51841bCF97421CDdb52d) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
+| Swap     | [`5.4.7`](https://www.npmjs.com/package/@airswap/swap/v/5.4.7)     | [`0x4572f2554421Bd64Bef1c22c8a81840E8D496BeA`](https://etherscan.io/address/0x4572f2554421Bd64Bef1c22c8a81840E8D496BeA) | [`953956f`](https://github.com/airswap/airswap-protocols/commit/953956f308c65ec53d1f1b20d35f47fe04b936af) |
+| Indexer  | [`3.6.8`](https://www.npmjs.com/package/@airswap/indexer/v/3.6.8)  | [`0xbA9aB9710Bd461F30C247f4cA2Cb7f453C22570e`](https://etherscan.io/address/0xbA9aB9710Bd461F30C247f4cA2Cb7f453C22570e) | [`5e8a07c`](https://github.com/airswap/airswap-protocols/commit/5e8a07c7cb7d2b9a8456194eeb43da853b21f45f) |
```
Comment (alicevhenshaw) [inline instant/contract-deployments.md]: At the provided commit, the indexer version is 2.6.7.beta.2 not 3.6.8

Comment (alicevhenshaw) [reply instant/contract-deployments.md]: I think the commit and version we provide should match

File: instant/contract-deployments.md (hunk 2 above)
Comment (poiresel) [reply instant/contract-deployments.md]: they've been updated.

File: instant/contract-deployments.md (hunk 1 above)
Comment (poiresel) [reply instant/contract-deployments.md]: nice catch!

File: instant/contract-deployments.md (hunk 3 above)
Comment (poiresel) [reply instant/contract-deployments.md]: updated it to be https://github.com/airswap/airswap-protocols/commit/48da430ca23876b9d01743d59b548f7fb59660ab

**Pull request merged**